plugins/thunderbolt/fu-thunderbolt-image.c

```diff
@@ -434,6 +456,9 @@ get_device_locations (guint16 id)
 	case 0x15D3:
 	case 0x15DA:
 	case 0x15C0:
+	case 0x15E7:
+	case 0x15EA:
+	case 0x15EF:
```

Comment:
Oops, I forgot that this location isn't relevant for TR devices anymore. The "X of N" info is now part of the DROM. Let's discuss it.

Reply:
@ybernat so if just this part of PR is dropped is it OK then?  Or is this a more involved change?

Reply:
No, it needs to add a new function to validate the "X of N" info from the DROM for those devices.
If preferred, it's possible to split it to 2 PRs. First add only the TR host support from this PR (by removing the IDs from this function, so it returns `NULL` and fails the validation) and then add the TR device support on a later PR.

Reply:
Yeah I'd like to see TR host in 1.0.6 if possible, so splitting the two makes sense to me. 
I think once @hughsie has the capsule cleanup PR done (and probably baking for a few days) he wants to cut 1.0.6.  If @finikorg can get the second PR for TR device done and tested before then, even better :)

Reply:
done